### ISSUE TITLE
You: bad_domain_for_stytch_sdk

### ISSUE THREAD

**Issue by scuty2000** [xtekky/gpt4free#1824]
**Bug description**
You.com no longer works (both on docker UI and python library)

**Screenshots**
![image](https://github.com/xtekky/gpt4free/assets/22243133/36b7a008-173c-4edf-8c6c-46f90fc56f46)

**Snippets**

Python code:
```python
response = g4f_client.chat.completions.create(
                model=g4f.models.claude_3_opus,
                messages=[{"role": "user", "content": prompt}],
                proxy="socks5://redacted:redacted@p.webshare.io:80"
            )
```

Error:
`Response 400: {"status_code":400,"error_type":"bad_domain_for_stytch_sdk","error_message":"This website has not been registered as an allowed domain for the Stytch SDK. Please add it here: https://stytch.com/dashboard/sdk-configuration","error_url":"https://stytch.com/dashboard/sdk-configuration"}`

**Environment**
- python version: 3.11.5
- g4f version: 0.2.9.4
- location:  Italy

@hlohaus

**Comment by leezhuuu** [comment]
+1

**Comment by scuty2000** [comment]
This issue as been solved, update to 0.2.9.5 with
`pip install -U g4f` for the python library
`docker pull hlohaus789/g4f:latest` for the docker
